joomlatools/joomlatools-pages
# Feature/321 gitpod

This PR closes #321 

## Support question raised: 
https://github.com/gitpod-io/gitpod/issues/1487

To see whether it is possible to preconfigure the editor to use multiple workspace roots. 


## Testing notes: 

* Install the gitpod browser extension:
https://www.gitpod.io/docs/browser-extension/

* Open the PR for this development work: 
https://github.com/joomlatools/joomlatools-pages/pull/325

* Click the newly created corresponding `gitpod` button 

* Click login with github

* You will see `Pulling Docker Images`

* On the next screen (workspace starting) you should see a load of content scrolling through the `_Terminal` tab bottom of the screen 

* When you see `Commands you can try:` appear the site is ready

* Look to the bottom right hand corner you will see `A service is available on port 8080... Click `Open browser` 

* Ensure that the editor explorer view is used Explorer 

	View > Explorer  use these shortcuts ⇧ ⌘ E

* Make some changes to the `content` folder` 

* Refresh the browser window do you see the changes
